examples/hpc/hello-dla/.cargo/config.toml

```diff
@@ -1,6 +1,9 @@
 [target.riscv64imac-unknown-none-elf]
 runner = "../../../scripts/run_on_hpc.sh"
 rustflags = ["-C", "link-arg=-Tmem_hpc.x", "-C", "link-arg=-Tlink.x"]
+# Uncomment to run on SDRAM
+#linker = "riscv64-unknown-elf-ld"
+#rustflags = ["-C", "link-arg=-Tsdram_hpc.x", "-C", "link-arg=-Tlink.x"]
```

Comment:
Should we use this linker for all HPC builds? Why is the linker sdram-specific?

Reply:
We could move rustflags into Justfile and out of .cargo/config.toml, as these link scripts are no longer the sensible default. Then have separate tasks for each. But that's a low priority thing. I can take care of it if I stumble on it.

Reply:
The linker is used to avoid issues related to linking to addresses over 32-bits so it's only necessary for SDRAM builds, as opposed to the linker that's backed with cargo/rustup/riscv64-unknown-none-elf target. The reason we don't use it for all builds is that we would have to download it twice during CI/CD. Passing it as a flag sounds like a good idea.